appium-tests/common/common.spec.js

```diff
@@ -114,14 +116,15 @@ describe('Contacts Android', function () {
             })
             .click()
             .context(webviewContext)
-            .executeAsync(function (pID, cb) {
+            .executeAsync(function (pID, getOnlyBirthday, cb) {
```

Comment:
Yeah, instead of Date object Appium gets just empty object in the full contact. 
I haven't been able to find it in Appium docs so I'm assuming it's just an undocumented quirk.